crypto/bn/bn_lib.c

```diff
@@ -426,11 +426,11 @@ int BN_set_word(BIGNUM *a, BN_ULONG w)
     return 1;
 }
 
-typedef enum {BIG, LITTLE} endianess_t;
+typedef enum {BIG, LITTLE} endianness_t;
```

Comment:
"typos" in code, such as this, should generally be fixed separately to obvious typos in comments and documentation

Reply:
Reverted.